libbuteosyncfw/profile/SyncSchedule.h

```diff
@@ -106,6 +118,7 @@ class SyncSchedule
      * \return Set of week day numbers.
      */
     DaySet days() const;
+    QVariantList daysAsVariant() const;
```

Comment:
Think this could be cleaner with just one getter returning a flagged int for the days. QSet feels like an overkill for such a thing, especially if it needs a separate thing for qml.

The Qt day enum is not fit for flags, but being such a simple thing, could just define a separate enum?

Reply:
> Think this could be cleaner with just one getter returning a flagged int for the days.

Are you suggesting to break the API here ?

> could just define a separate enum?

I agree that would be much more simple with something like that:
```
    enum Day {
        NoDays    = 0x00,
        Monday    = 0x01,
        Tuesday   = 0x02,
        Wednesday = 0x04,
        Thursday  = 0x08,
        Friday    = 0x10,
        Saturday  = 0x20,
        Sunday    = 0x40
    };
    Q_DECLARE_FLAGS(Days, Day)
    Q_FLAG(Days)

    Days days() const;
```
Is it what yoy have in mind ?